site/content/contribute/mobile/developer-setup/troubleshooting.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Troubleshooting"
+date: 2017-08-20T11:35:32-04:00
+weight: 1
+subsection: Developer Setup
+---
+
+### Errors When Running 'make run-android'
+
+##### Error message
+```sh
+React-native-vector-icons: cannot find dependencies
+```
+
+##### Solution
+Make sure the **Extras > Android Support Repository** package is installed with the Android SDK.
+
+#### Error message
+```sh
+Execution failed for task ':app:packageAllDebugClassesForMultiDex'.
+> java.util.zip.ZipException: duplicate entry: android/support/v7/appcompat/R$anim.class
+```
+
+#### Solution
+Clean the Android part of the mattermost-mobile project. Issue the following commands:
+
+1. ``cd android``
+2. ``./gradlew clean``
+
+##### Error message
+```sh
+Execution failed for task ':app:installDebug'.
+> com.android.builder.testing.api.DeviceException: com.android.ddmlib.InstallException: Failed to finalize session : INSTALL_FAILED_UPDATE_INCOMPATIBLE: Package com.mattermost.react.native signatures do not match the previously installed version; ignoring!
+```
+
+##### Solution
+The development version of the Mattermost app cannot be installed alongside a release version. Open ``android/app/build.gradle`` and change the applicationId from ``"com.mattermost.react.native"`` to a unique string for your app.
+
+#### Error Message
+```
+[Error] Error: Compilation of µWebSockets has failed and there is no pre-compiled binary available for your system. Please install a supported C++11 compiler and reinstall the module 'uws'.
+... looping infinitely ...
+```
+##### Solution
+
+Your most likely using the wrong version of npm. Recommended is 9.3.0.
+
+### Errors When Running 'make run-ios'
+
+##### Error message
+```sh
+xcrun: error: unable to find utility "instruments", not a developer tool or in PATH
+```
+
+##### Solution
+
+- Launch XCode and agree to the terms first.
+- Go to **Preferences -> Locations** and you'll see an option to select a version of the Command Line Tools. Click the select box and choose any version to use.
+
+![](/img/mobile/xcode_preferences.png)
```

Comment:
This image is not showing in the preview

Reply:
Looks fine to me
![image](https://user-images.githubusercontent.com/6757047/54536926-66b7d800-4970-11e9-9247-fcd20427861f.png)